google/cloud/security/scanner/scanners/instance_network_interface_scanner.py

```diff
@@ -0,0 +1,170 @@
+        network_interfaces = []
+        for instance in instances:
+            network_interfaces += instance.create_network_interfaces()
+        return network_interfaces
+
+    def parse_instance_network_instance(self, instance_object):
+        """Create a list of network interface obj."""
+        return instance_object.create_network_interfaces()
+
+    def _get_project_policies(self):
+        """Get projects from data source."""
+        project_policies = {}
+        project_policies = (
+            project_dao.ProjectDao().get_project_policies('projects',
+                                                          self.
+                                                          snapshot_timestamp))
+        return project_policies
+
+    @staticmethod
+    def _get_resource_count(project_policies, instance_network_interfaces):
+        """Get resource count for org and project policies.
+
+        Args:
+            project_policies: dict containing the projects
+                (gcp_type.project.Project) and their iam policies (dict).
+            instance_network_interfaces: list of network_interface objects. 
+        Returns:
+            Resource count map
+        """
+        resource_counts = {
+            ResourceType.PROJECT: len(project_policies),
+            ResourceType.INSTANCE: len(instance_network_interfaces),
+        }
+
+        return resource_counts
+
+    def _retrieve(self):
+        """Run the data collection."""
+        enforced_networks_data = []
+        #project_policies = {}
+        instance_network_interfaces = self.get_instance_networks_interfaces()
+        #enforced_networks_data.append(instance_network_interfaces)
+        #enforced_networks_data.append(project_policies)
+
+        #resource_counts = self._get_resource_count(project_policies,
+        #                                           instance_network_interfaces)
+
+        return instance_network_interfaces #enforced_networks_data #, resource_counts
+
+    def _find_violations(self, enforced_networks_data):
+        """Find violations in the policies.
+
+        Args:
+            enforced_networks_data: Enforced networks data
+            to find violations in
+            rules_engine: The rules engine to run.
+
+        Returns:
+            A list of violations
+        """
+        all_violations = []
+        LOGGER.info('Finding enforced networks violations...')
+        for instance_network_interface in enforced_networks_data:
+            LOGGER.debug('%s', instance_network_interface)
+            violations = self.rules_engine.find_policy_violations(
+                instance_network_interface)
+            LOGGER.debug(violations)
+            all_violations.extend(violations)
+        return all_violations
+
+    def run(self):
+        """Runs the data collection."""
+        instance_network_interface_data = self._retrieve()
+        all_violations = self._find_violations(instance_network_interface_data)
```

Comment:
Readability: Fix the line wrap please.

You can do

all_violations = (
    self._find_violations(instance_network_interface_data))

for example.